apps/public-docsite-v9/package.json

```diff
@@ -30,6 +32,7 @@
     "@griffel/react": "^1.4.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-window": "^1.8.6",
```

Comment:
Virtualizing the icons catalog so it doesn't take aaages to search anything. Perhaps we should do the same thing with our official icons list ? @tomi-msft